### DISCUSSION TITLE
React application build size getting increased from 12 MB to 86 MB for fluent version 9

### DISCUSSION THREAD

**Discussion by hclhims** [microsoft/fluentui#30538] [Q&A]
Hi Team,

Need urgent help on the build size issue which is getting increased for my react application when i am trying to use fluent v9 controls. 
So when i tried to add fluent version 9 using below yarn command
yarn add @fluentui/react-components
So once i build my application then build size gets increased from 12 MB to 86 MB and due to which i am not able to publish this application in figma plugin as the restricted size for plugin is only 15 MB

Kindly help as i am stuck over here

**Comment by tombohub** [comment]
We are gonna need more information. 

What react version?

Are you using some frameworks like next or remix?

Are you using Vite?

**Reply by hclhims** [reply]
Hi,

Thanks for your response. Below is the required information:

React Version : 17.0.2

Fluent Version : 9

No frameworks like Next or Remix

No we are not using Vite

**Reply by tombohub** [reply]
What are you using for bundling?

**Reply by hclhims** [reply]
Using webpack for bundling
npx webpack --mode production
npx webpack --mode development

**Reply by tombohub** [reply]
I am sorry, I know very little about webpack. Did you try to use visualizer: https://github.com/jonamat/webpack-visualizer-plugin2#readme ? Post here lets see what takes all that space

**Reply by hclhims** [reply]
Thanks for response. I have already tried webpack bundle analyser and found out that @fluentui/react-components is increasing the bundle size
